pkg/filetree/filetree_test.go

```diff
@@ -786,3 +786,94 @@ func TestFileTree_File_CycleDetection(t *testing.T) {
 	}
 
 }
+
+func TestFileTree_AllFiles(t *testing.T) {
+	tr := NewFileTree()
+
+	paths := []string{
+		"/home/a-file.txt",
+		"/sym-linked-dest/a-.gif",
+		"/hard-linked-dest/b-.gif",
+	}
+
+	for _, p := range paths {
+		_, err := tr.AddFile(file.Path(p))
+		if err != nil {
+			t.Fatalf("failed to add path ('%s'): %+v", p, err)
+		}
+	}
+
+	var err error
+
+	// dir
+	_, err = tr.AddDir("/home")
+	if err != nil {
+		t.Fatalf("could not setup link: %+v", err)
```

Comment:
What does "could not setup link" mean here?

Reply:
should say dir, I'll adjust